cypress/integration/login.js

```diff
@@ -16,7 +16,7 @@ describe('Login functionality', function() {
 
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
+        cy.get('.calendar__day').contains('14');
```

Comment:
It is strange that this test case passing, as today is the 13th. But it seems that you are asserting "calendar_day" and it might be picking up all days in the calendar and not specifically today's date